### DISCUSSION TITLE
Is this working on Android 14 Update? Edit: It's working.

### DISCUSSION THREAD

**Discussion by andresro14** [kyujin-cho/pixel-volte-patch#86] [General]
**Update: It is working on June 2023 Update.**

Hi.
I'm asking this because this is really important to me and I'm wondering if the patch is working on March update.
Someone can confirm if it's working on stable?

cc @kyujin-cho

**Comment by adrian-cocorev** [comment]
Just did. Had to re-apply the patch, but it works. (Romania)

**Reply by MariusRO1** [reply]
Which model and which cartier?

**Reply by adrian-cocorev** [reply]
P7Pro, Orange (yoxo). Btw, forgot to mention, I've used an older version of the patch. One that I already had, v1.2.2

**Reply by MariusRO1** [reply]
All the addons are working?(voLTE, voWIFI)

**Reply by adrian-cocorev** [reply]
Yes. Noticed that sometimes if switching back an forth from LTE to Wifi, it doesn't call right away, but it's not that often

**Reply by MariusRO1** [reply]
Hi. It is working after April update?

**Reply by adrian-cocorev** [reply]
Hi. Yep. Had to reapply the patch again, v1.2.2, but it's working

**Reply by MariusRO1** [reply]
What about after may update?😀

**Reply by andresro14** [reply]
> What about after may update?😀

It's working on June update :)

**Reply by adrian-cocorev** [reply]
Doesn't work me as VoLTE is kinda now supported in Romania now. No VoWifi though, and VoWifi is what needed. Tried multiple times, w/o succes.

**Comment by andresro14** [comment]
I updated after your comment and it's working!

**Comment by lupusthunder** [comment]
hello, not working with v1.2.3 
nvm, found v1.2.2 Volte works again but Vowifi doesn't

thnx !

**Comment by andresro14** [comment]
Just updated to Android 14 and it's still working 😎

**Comment by Tintolios** [comment]
working 100% android 14

![Screenshot_20231005-031634-EDIT](https://github.com/kyujin-cho/pixel-volte-patch/assets/65043538/5759cfb7-bafc-436a-b4d2-39a66e041ded)
![Screenshot_20231005-031852](https://github.com/kyujin-cho/pixel-volte-patch/assets/65043538/81825322-31c4-40ca-bba9-4ebcc59787c1)

**Comment by bige93** [comment]
Looks like Backup Calling toggle is not there where I left it in Android 13. Not sure if Google people moved it or what happened. 

Could someone please check?

![Screenshot_20231007-161549](https://github.com/kyujin-cho/pixel-volte-patch/assets/66921212/762c1a3b-0c48-4a84-b60f-010cd70ecba9)
![Screenshot_20231007-161351](https://github.com/kyujin-cho/pixel-volte-patch/assets/66921212/ec1a9773-f468-4b98-898d-6fe7771e0160)

**Comment by clntmamuri** [comment]
Is it still working on Android 14 January 2024 update?

**Reply by RobertLaffan** [reply]
I just updated to the latest beta update for Pixel 8 Pro (QPR2 Beta 3 AP11.231215.007), and my app is now crashing as I start it. After a restart, it comes up briefly to the screen that asks for permissions, then crashes there.

**Reply by clntmamuri** [reply]
Same thing as mine. It crashes when opening Pixel IMS.

**Reply by RobertLaffan** [reply]
There is a modified version you can use that works, from the ISSUES thread on GitHub.

**Reply by clntmamuri** [reply]
Thanks! It works.
